hi-ml-cpath/src/health_cpath/utils/plots_utils.py

```diff
@@ -116,11 +124,11 @@ def save_confusion_matrix(results: ResultsType, class_names: Sequence[str], figu
         true_labels,
         pred_labels,
         labels=all_potential_labels,
-        normalize="true"
+        normalize=normalize,
     )
 
     fig = plot_normalized_confusion_matrix(cm=cf_matrix_n, class_names=(class_names))
-    save_figure(fig=fig, figpath=figures_dir / f"normalized_confusion_matrix_{stage}.png")
+    save_figure(fig=fig, figpath=figures_dir / f"{'normalized_' if normalize else ''}confusion_matrix_{stage}.png")
```

Comment:
as we discussed earlier, perhaps we could have one matrix showing both normalized and non-normalized values?

Reply:
Yes, on it!